CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+# 4.1.0 / 2019-01-20
+
+* [FEATURE] Fail with explicit message if OS is not supported by the role. See [#247][]
+* [BUGFIX] Ensure that system-probe is stopped if it is disabled or not installed. See [#249][]
+* [BUGFIX] Change default datadog_agent group to dd-agent. See [#248][]
+* [OTHER] Update instructions to use datadog.datadog as the role name. See [#246][]
+* [OTHER] Add development guidelines & small kitchen dev environment. See [#243][]
```

Comment:
These last two points aren't really "released" in 4.1.0.

Reply:
That's true, they're docs changes, but we've been listing them in the past, so I kept them

Reply:
I'm fine both ways.